hera_mc/tests/test_utils.py

```diff
@@ -1,8 +1,39 @@
 import nose.tools as nt
-
+from astropy.time import Time
 from .. import utils
+sidesec = 365.25/366.25 #length of sidereal second in SI seconds.
 
-
+def test_LSTScheduler_lstbinsize():
+    """
+    test that two bins have the right time separation
+    """
+    LSTbin_size = 10
+    starttime1 = Time('2019-9-19T05:05:05.0', format='isot', scale='utc')
+    scheduletime1, hour1 = utils.LSTScheduler(starttime1, LSTbin_size)
+    starttime2 = Time('2019-9-19T05:05:15.0', format='isot', scale='utc')
+    scheduletime2, hour2 = utils.LSTScheduler(starttime2, LSTbin_size)
+    # print("difference between time bins")
+    # print("start difference (should be 15):",(starttime2-starttime1).datetime.seconds)
+    # print("lst difference between calculated start times (should be 10s):",hour2-hour1)
+    # print("utc difference (should be 9.9ish):",(scheduletime2-scheduletime1).value*24*3600)
```

Comment:
please remove these print statements and the ones below.

Reply:
done